### PR TITLE
feat: add `coder stat` command

### LINKED ISSUE #7076: agent metadata: add `coder stat` to identify Kubernetes cgroup usage
Running `free` or `top` inside a container does not give accurate usage info and reading cgroup values, particularly for CPUs can be tricky. A `coder stat` command can help help

- [ ] finish `coder stat` for CPU, DIsk, Memory in Docker and Kubernetes containers
- [ ] add to our Docker and Kubernetes example templates

### PR DESCRIPTION
**Pull request by ammario**
closes #7076

### CODE REVIEW

**Comment by ammario** [comment]
@mafredri the `stat` name (by itself) is not great. I was trying to be consistent with `dstat`, `vmstat`, `sysstat` and friends. We could follow in the footsteps of `dstat` and make it `cstat`? I worry that `info` or `stats` could be confused for a command that gets the stats of a workspace. I'm also hesitant to use a long name because this invocation will be repeated frequently in templates.

**Comment by mafredri** [comment]
@ammario fair. Given that, I don't really have any great suggestions or alternatives. Calling it `sysstat` would be quite good, but wouldn't want it to be confused with the actual suite of tools. Likewise `cstat` is good but doesn't inherently mean much to the casual user. I guess that leaves `stat` or `cstat` as options, or if you manage to come up with a better one 😄

**Comment by ammario** [comment]
@mafredri sysstat is my favorite as well. Might just steal the name.

**Comment by ammario** [comment]
I'm not going to have time to work on this in the next couple of weeks. @bpmct may assist in finding someone new.

File: cli/stat.go [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+		groupDir = fields[2]
+		break
+	}
+
+	if groupDir == "" {
+		return 0, xerrors.New("no cpu cgroup found")
+	}
+
+	cpuAcct := func() (int64, error) {
+		path := fmt.Sprintf("/sys/fs/cgroup/cpu,cpuacct/%s/cpuacct.usage", groupDir)
```
Comment (matifali) [inline cli/stat.go:93]: @bpmct go pkg doesn't handle this on its own, and we need to specify the path of the cgroup, which is kernel/release dependent.

**Pull request closed without merging**